### PR TITLE
Enable injecting RPC via URL

### PR DESCRIPTION
**Pull request by oskarvu**
Closes FRO-766

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +108,15 @@ This will stop recording requests and use the HAR file instead.
 - You can now run the test again and potentially fix assertions that failed in the previous run.
 - If necessary, you can modify the HAR file manually. This might be useful when you need to test rare edge cases.
 
+### Injecting rpc through url
+
+In development and staging environments, there is an option to inject custom `rpc` with `chainId` via url.
```
Comment (krzkaczor) [inline README.md:113]:
```suggestion
In development and staging environments, there is an option to inject custom `rpc` with `chainId` via url. To instrumenting the app to test fork environments etc.
```

File: packages/app/src/config/wagmi/getTransports.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export interface GetTransportsParamsOptions {
+  injectedNetwork?: InjectedNetwork
+  forkChain?: Chain
+}
+
+export type GetTransportsResult = Record<SupportedChainId, Transport>
+
+export function getTransports({ injectedNetwork, forkChain }: GetTransportsParamsOptions): GetTransportsResult {
+  const injectedMainnetTransport = injectedNetwork?.chainId === mainnet.id ? http(injectedNetwork.rpc) : undefined
```
Comment (krzkaczor) [inline packages/app/src/config/wagmi/getTransports.ts:17]: i think it will be much cleaner to write it as:
1: define basic transports
2. if injectedNetwork -> transports[injectedChainId]  = injectedTranport
3. if forkChain -> transports[forkChainId]...

**Review by krzkaczor — APPROVED** [review]

**Review by krzkaczor — APPROVED** [review]

**Review by yivlad — APPROVED** [review]
